aries-test-harness/features/0023-did-exchange.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Establishing DID Based Relationships RFC 0023
+
+   @T001-RFC0023 @P1 @critical @AcceptanceTest @RFC0023 @wip
+   Scenario: Establish a DID based connection between two agents with an explicit invitation
```

Comment:
The title of this is not great.  The connection is "DID based" regardless of whether 0160 of 0023 DID Exchange is used.  I'd recommend being more explicit about saying this is using 0023 DID Exchange (assuming it is).

Reply:
Title and other "DID based" references changed to DID Exchange explicitly. Also modified the final check for a successful connection to be a common step between both Connection protocol and DID Exchange. Unfortunately the states in the RFC for these two are different so the existing check of "complete" could not be used.